README.md

```
@@ -8,10 +8,18 @@ To include InjiVcRenderer in your Swift project:
 - Add package dependency: Enter Package URL of InjiVcRenderer repo
 
 ### API
-- `generateCredentialDisplayContent(credentialFormat: CredentialFormat, wellKnownJson: String? = nil, vcJsonString: String): [Any]` - expects the Verifiable Credential as parameter and returns the replaced SVG Template.
+- `generateCredentialDisplayContent(credentialFormat: CredentialFormat, wellKnownJson: String? = nil, vcJsonString: String, qrCodeData: String? = nil): [Any]` - expects the Verifiable Credential as parameter and returns the replaced SVG Template.
     - `vcJsonString` - VC Downloaded in stringified format.
     - `wellKnownJson` - Well-known Json downloaded in stringified format. It is optional parameter.
     - `credentialFormat` - Enum to specify the credential format. Currently only LDP_VC format is supported.
+    - `qrCodeData` (optional) - Custom data used to generate the QR code.
+       -  If provided → QR code is generated using this value.
+       -  If not provided or empty → entire VC JSON string is used as fallback.
+##### Custom QR Code Data Support
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h4; Actual: h5

(MD001, heading-increment)